ui/build.gradle

```diff
@@ -68,14 +68,14 @@ buildscript {
 
 dependencies {
 
-
+    implementation files('libs/securestorage.jar')
```

Comment:
Can you comment on why we've moved this to be a local jar? I notice the library hasn't been updated in a couple of years, has it been abandoned? Should we be planning to remove it?

Reply:
The artifacts aren't in maven anymore, I couldn't download them at all, so I had to get a jar.
This jar though is an updated version of the previous artifact from maven.

Reply:
Ok, I've made a Jira ticket to replace that at a later date (DDN-139)